google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -714,6 +714,99 @@ def create_file file, path = nil, acl: nil, cache_control: nil,
         alias_method :new_file, :create_file
 
         ##
+        # Concatenates a list of existing files in the bucket into a new file in
+        # the bucket. There is a limit (currently 32) to the number of files
+        # that can be composed in a single operation.
+        #
+        # #### Customer-supplied encryption keys
+        #
+        # By default, Google Cloud Storage manages server-side encryption keys
+        # on your behalf. However, a [customer-supplied encryption key](https://cloud.google.com/storage/docs/encryption#customer-supplied)
+        # can be provided with the `encryption_key` option. If given, the same
+        # key must be provided to subsequently download or copy the file. If you
+        # use customer-supplied encryption keys, you must securely manage your
+        # keys and ensure that they are not lost. Also, please note that file
+        # metadata is not encrypted, with the exception of the CRC32C checksum
+        # and MD5 hash. The names of files and buckets are also not encrypted,
+        # and you can read or update the metadata of an encrypted file without
+        # providing the encryption key.
+        #
+        # @param [Array<String, Google::Cloud::Storage::File>] sources
+        # @param [String, Google::Cloud::Storage::File] destination
+        # @param [String] content_type The
```

Comment:
If `compose` always creates a new file, then `destination` should always be a string, right?

Reply:
Yes, correct, thanks for catching this error.

Reply:
Ugh, i left that comment on the wrong line. Apologies.